packages/components/dialog/src/dialog.scss

```diff
@@ -21,7 +21,7 @@ $md-breakpoint: 600px;
   --_subtitle-font: var(--sl-text-dialog-subheading);
   --_subtitle-padding-bottom: var(--sl-space-dialog-subtitle-padding-bottom);
   --_title-font: var(--sl-text-dialog-heading);
-  --_body-font: var(--sl-text-body-lg-normal);
+  --_body-font: var(--sl-text-body-md-normal);
```

Comment:
This one was made like in Figma before, maybe it should be changed in Figma as well to have it consistent?

<img width="1249" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/139ba0bc-be3a-4cdd-be4d-b268ec1a451e">

Reply:
@DanielleRameau See Anna comment above; can you update Figma?